attendance/views.py

```diff
@@ -42,7 +42,16 @@ def user_check(request, username):
         return True
     else:
         return False
-
+        
+def last_month(this_month=datetime.now().month):
+	if(this_month > 0 and this_month <= 12):
+		if(this_month==1):
+			return 12
+		else:
+			return this_month-1
+	else: 
+		return "Number of month out of range"
```

Comment:
You should raise some kind of exception here -  (http://stackoverflow.com/questions/2052390/manually-raising-throwing-an-exception-in-python) `ValueError` would be probably best